indy_client/test/cli/test_send_schema.py

```diff
@@ -1,10 +1,24 @@
+import json
+
+from indy.anoncreds import issuer_create_schema
+from indy.ledger import build_schema_request
 from indy_client.test.cli.constants import SCHEMA_ADDED, SCHEMA_NOT_ADDED_DUPLICATE
+from indy_node.test.api.helper import sdk_write_schema
+from plenum.test.helper import sdk_get_reply, sdk_sign_and_submit_req
 
 
-def test_send_schema_multiple_attribs(be, do, poolNodesStarted, trusteeCli):
-    be(trusteeCli)
-    do('send SCHEMA name=Degree version=1.0 keys=attrib1,attrib2,attrib3',
-       expect=SCHEMA_ADDED, within=5)
+def testSendSchemaMultipleAttribs(looper, sdk_pool_handle, sdk_wallet_trust_anchor):
+
+    wallet_handle, identifier = sdk_wallet_trust_anchor
+
+    schema_json, _ = sdk_write_schema(looper, sdk_pool_handle, sdk_wallet_trust_anchor)
+    _, schema_json = looper.loop.run_until_complete(
+        issuer_create_schema(identifier, "name=Degree", "1.0", json.dumps(["attrib1","attrib2","attrib3"])))
+
+    request = looper.loop.run_until_complete(build_schema_request(identifier, schema_json))
+
+    return schema_json, \
```

Comment:
Each test must end with an assertion, not return.
Ideally, each test should be flat, straightforward, and consist of 3 main parts:
1) Configuration 
2) Execution
3) Assertion